release-tools/metabuild.sh

```diff
@@ -52,6 +52,19 @@ add_makefile_prog() {
 	fi
 }
 
+get_cpusupport_cflags() {
+	src=$1
+
+	# Get the flags from the .c file and format them.
+	str=$(grep 'CPUSUPPORT CFLAGS:' $S | cut -f 2- -d :)
+	cflags_unstripped=$(for X in ${str} ; do		\
```

Comment:
This seems unnecessarily complicated.  Why not just

```
for X in ${str}; do
    printf " \${CFLAGS_%s}" "$X";
done | sed 's/^ //'
```

and omit the `cflags_unstripped` and `cpusupport_cflags` variables entirely?